src/api/c/wrap.cpp

```diff
@@ -19,11 +19,10 @@ using af::dim4;
 using namespace detail;
 
 template<typename T>
-static inline void wrap(af_array* out, const af_array in, const dim_t ox,
-                        const dim_t oy, const dim_t wx, const dim_t wy,
-                        const dim_t sx, const dim_t sy, const dim_t px,
-                        const dim_t py, const bool is_column) {
-    wrap<T>(getArray<T>(*out), getArray<T>(in), ox, oy, wx, wy, sx, sy, px, py,
+static inline void wrap(af_array* out, const af_array in, const dim_t wx,
```

Comment:
We converted those functions to include the output parameter instead of returning output as a return value. We can just query for those values form the out Array<T> so they are not required anymore.